### ISSUE TITLE
where is lambda_w and lambda_w+?

### ISSUE THREAD

**Issue by 37man** [daicoolb/PHDMF#5]
It's a great pleasure to read your paper, and it's a great job. But I have some questions to ask you. I notice that there are lambda_w and lambda_w+ parameters in loss function of PHD. but I can not find them in your code.  where are they in your code? and how did you set them? why your paper did not show how they affect your model?

**Comment by daicoolb** [comment]
sorry, now i didn‘t remember the deatils of the code, maybe you can see the training steps.
